distribution/docs/src/main/resources/content/_running/manual-removal-expired.adoc

```diff
@@ -0,0 +1,38 @@
+:title: Manual Removal of Expired Records
+:type: dataManagement
+:status: published
+:summary: Methods of removing expired records from the Catalog.
+:parent: Removing expired records from the Catalog
+:order: 00
+
+== {title}
+
+To manually remove expired records from the Catalog, execute in the ${command-console}:
+
+----
+catalog:removeall --expired
+----
+
+When prompted, type `yes` to remove all expired records.
+
+[TIP]
+====
+For help on the `removeall` command, execute
+
+`help catalog:removeall`
+====
+
+The `catalog:removeall` command states you want to remove records from the local Catalog.
+
+The `--force` option removes without a confirmation message.
+
+The `--expired` option is to remove only expired records.
+
+[IMPORTANT]
+====
+If the `--expired` option is omitted, then all records will be removed from the Catalog.
+====
+
+The Catalog Command is convenient for removing records but it is not guaranteed to be fast or efficient. Additionally, certain configurations for archiving Catalog records can dramatically slow the process. If efficiency and speed are desired, interact directly with the catalog storage provider (e.g. Solr, Oracle) to delete records.
```

Comment:
✏️ if you go through here again might be worth removing `Oracle` references

Reply:
dropped